modules/ckanext-ytp_main/ckanext/ytp/reports.py

```diff
@@ -56,7 +56,7 @@ def with_totals(orgs):
 
     # Optimization opportunity: Prefetch datasets for all related orgs in one go
     root_datasets_pairs = (
-        (k, list(package_generator('owner_org:(%s)' % ' OR '.join(v), 1000, context)))
+        (k, list(package_generator('owner_org:(%s)' % ' OR '.join(v), 10, context)))
```

Comment:
This fetches all packages to memory anyway, so a smaller page size doesn't help here.